pkg/cloud_provider/file/file.go

```diff
@@ -646,6 +658,99 @@ func (manager *gcfsServiceManager) StartResizeShareOp(ctx context.Context, obj *
 	return nil, nil
 }
 
-func (manager *gcfsServiceManager) WaitForOpWithOpts(ctx context.Context, opts PollOpts) error {
+func (manager *gcfsServiceManager) WaitForOpWithOpts(ctx context.Context, op string, opts PollOpts) error {
+	return nil
+}
+
+func (manager *gcfsServiceManager) GetOp(ctx context.Context, op string) (*filev1beta1.Operation, error) {
```

Comment:
same pattern in `waitForOp` above, could substitute with this method

Reply:
getop only does GET call for op.

WaitForOpWithOpts will do a get and poll for complete sttaus.

Reply:
Not talking about `WaitForOpWithOpts` but `waitForOp` in line 495.

Reply:
Not talking about `WaitForOpWithOpts` but `waitForOp` in line 495.